server/bundler/css-hot-reload.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Module dependencies
+ */
+var os = require( 'os' ),
+	fs = require( 'fs' ),
+	path = require( 'path' ),
+	crypto = require( 'crypto' ),
+	spawn = require( 'child_process' ).spawn,
+	debug = require( 'debug' )( 'css-hot-reload' ),
+	chalk = require( 'chalk' ),
+	chokidar = require('chokidar');
+
+/**
+ * Initialize server CSS hot-reloading logic
+ */
+function setup( io ) {
+
+	const SCSS_PATHS = [ './assets/stylesheets/', './client/' ];
```

Comment:
@dmsnell This evening when I get home I'll try and setup my win laptop for calypso again and do some tests, last time I tried setting it up it completely screwed up my MinGW and Git Bash installation, hopefully this time it will be a success! :8ball: